### PR TITLE
Fix #761: Invalid presence check limit check

### LINKED ISSUE #761: Invalid presence check limit check
The presence check limit is checked using SMS OTP count (default limit: 5). This is problematic, because the user can resend a SMS OTP, and then the number of presence check attempts does not correspond to the SMS OTP count. This manifests in the data that some users have up to 8 failed SMS OTPs.

We should consider implementing a different strategy, e.g. using a separate table which stores all presence check attempts and their results, including the SMS OTP result because it belongs to the same SCA verification step.

### PR DESCRIPTION
**Pull request by banterCZ**

### CODE REVIEW

File: docs/db/changelog/changesets/enrollment-server-onboarding/1.5.x/20230614-add-sca-result.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            <not>
+                <tableExists tableName="es_sca_result"/>
+            </not>
+        </preConditions>
+        <comment>Create a new table es_onboarding_process</comment>
+        <createTable tableName="es_sca_result">
+            <column name="id" type="bigint">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="identity_verification_id" type="varchar(36)">
```
Comment (romanstrobl) [inline docs/db/changelog/changesets/enrollment-server-onboarding/1.5.x/20230614-add-sca-result.xml:27]: I suggest we also add `process_id` for simpler joins.

File: docs/onboarding/PowerAuth-Enrollment-Onboarding-Server-1.5.0.md [hunk 2]
```diff
@@ -75,6 +75,53 @@ ALTER TABLE es_onboarding_process
...
 
+### SCA Result
+
+A new table `es_sca_result` has been created.
+
+
+#### PostgreSQL
+
+```sql
+CREATE SEQUENCE es_sca_result_seq INCREMENT BY 50 START WITH 1 CACHE 2;
```
Comment (romanstrobl) [inline docs/onboarding/PowerAuth-Enrollment-Onboarding-Server-1.5.0.md:86]: What is the point of caching 2 sequence IDs?

**Review by romanstrobl — APPROVED** [review]
Looks nice, I found two minor issues.

File: docs/onboarding/PowerAuth-Enrollment-Onboarding-Server-1.5.0.md (hunk 2 above)
Comment (banterCZ) [reply docs/onboarding/PowerAuth-Enrollment-Onboarding-Server-1.5.0.md:86]: Let's drop the explicit CACHE size and use the default.

File: docs/db/changelog/changesets/enrollment-server-onboarding/1.5.x/20230614-add-sca-result.xml (hunk 1 above)
Comment (banterCZ) [reply docs/db/changelog/changesets/enrollment-server-onboarding/1.5.x/20230614-add-sca-result.xml:27]: Ok, why not.

**Review by romanstrobl — APPROVED** [review]
Great, thanks!

**Pull request merged**